priv/worker.py

```diff
@@ -232,7 +247,7 @@ def process_response(response):
       envs.env_close(instance_id)
 
     instance_id = envs.create(enviroment)
-    data = json.dumps({"instance" : instance_id})
+    data = json.dumps({"instance" : instance_id},cls = NDArrayEncoder)
```

Comment:
Do you mind to use a space right after ``,``just to be conform with the rest of the code.

Reply:
Okay  done !!